connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectClusterAssertions.java

```diff
@@ -45,7 +45,7 @@
 
     private static final Logger log = LoggerFactory.getLogger(EmbeddedConnectClusterAssertions.class);
     public static final long WORKER_SETUP_DURATION_MS = TimeUnit.SECONDS.toMillis(60);
-    public static final long VALIDATION_DURATION_MS = TimeUnit.SECONDS.toMillis(30);
+    public static final long VALIDATION_DURATION_MS = TimeUnit.SECONDS.toMillis(5);
```

Comment:
Does this reduce test runtime? Might this introduce flakiness in a CI environment?

Reply:
I highly doubt it; validations usually take place very quickly. I lowered the value because I used test-driven development for this PR and at the beginning some of my failing test cases were talking 30 seconds each. If we really want to play it safe I think 10 seconds should be fine. Or, alternatively, we could revamp the validation assertions to not use `TestUtils::waitForCondition` at all and only issue one validation request.

Reply:
>  lowered the value because I used test-driven development for this PR and at the beginning some of my failing test cases were talking 30 seconds each.

Ah, this is a productivity win for everyone using the embedded assertions then. :+1:

> we could revamp the validation assertions to not use TestUtils::waitForCondition at all and only issue one validation request.

I don't think this is necessary, and any tests that rely on retries for validations would definitely flake out in CI.